modules/demand_electricity/workflow/envs/plot.yaml

```diff
@@ -0,0 +1,11 @@
+# Adding this because dependencies in geo.yaml are very old
+# And cause a problem with matplotlib=3.7.1
+# Once geo.yaml is updated, this can env can probably be removed
+name: plot
+channels:
+    - conda-forge
+dependencies:
+  - python=3.11
+  - pandas
+  - geopandas
+  - matplotlib
```

Comment:
please freeze the version of the libraries!

Reply:
Do we want to freeze all environments managed by snakemake? Seems ok, just have to remove all restrictions once in a while, re-run, fix all bugs that may appear and freeze again.